kolibri_explore_plugin/views.py

```diff
@@ -114,7 +114,7 @@ def get(self, request, app):
 class EndlessLearningCollection(View):
     COLLECTIONS = {
         "small": {
-            "title": "3Gb",
+            "title": "3GB",
```

Comment:
I hate to be a pedant but there should be a space between the number and the unit. If we are feeling really pedantic it should be a nonbreaking space or similar.

Reply:
Knowledge learned.
I have not known the rule until I found https://english.stackexchange.com/questions/2794/punctuation-with-units